Program.cs

```diff
@@ -1,16 +1,161 @@
 ﻿// See https://aka.ms/new-console-template for more information
 
+using System.Globalization;
+using System.Runtime.InteropServices.JavaScript;
 using Kristina_Kulich__Application;
+using Kristina_Kulich__Application.Helpers;
+using Kristina_Kulich__Application.CSharpLinq;
 
 class Program
 {
     static void Main(string[] args)
     {
-     new Homework_1().Task_1();   
-     new Homework_1().Task_2();
-     new Homework_1().Task_3();
-     new Homework_1().Task_4();
-     new Homework_1().Task_5();
-     new Homework_1().Task_6();
-     }
+        #region Task1
+
+        int[] integers = new[] { 12, 8, 27, 78, 2, 3, 3, 4, 21, 1 };
+        List<int> divedsby3 = integers.Select(i => i).Where(i => (i % 3 == 0)).ToList();
+        foreach (var v in divedsby3)
+        {
+            Console.WriteLine("Devides by 3 without remainder: {0}", v);
+        }
+
+        #endregion
+
+        #region Task2
+
+        int[] integers2 = new[] { 15, -8, -7, -5, 6, 14, 13, -9, 11, -10 };
```

Comment:
why there are 2 duplicated pieces of code?

Reply:
fixed